.github/workflows/linux.yml

```diff
@@ -741,6 +741,101 @@ jobs:
             ${{ env.INSTALL_TEST_DIR }}/TEST*.xml
           if-no-files-found: 'error'
 
+  Python_Unit_Tests_Model_Optimizer:
```

Comment:
why these tests are duplicated and not removed from common python job?

Reply:
because this is draft, please do not review for a while.